operations/research-and-development/product/release-process/desktop-release.md

```diff
@@ -2,22 +2,24 @@
 
 This document outlines the release process for the Mattermost desktop app.
 
-Desktop App releases follow a fixed schedule of 4 releases per year. Releases ship quarterly by the 16th of March, June, September, and December of each year.
+Desktop app releases follow a fixed schedule of 4 releases per year. Releases ship quarterly by the 16th of March, June, September, and December of each year.
 
 A dot release will be prepared sooner if [Electron](https://github.com/electron/electron/releases) releases a security update, or if other urgent bugs are found.
 
-Please refer to [the desktop app release playbook](https://community.mattermost.com/playbooks/playbooks/h3a39biacpnuim7ufmwiuuoxfo/outline) for a most up-to-date checklist.
+Please refer to [the desktop app release playbook](https://community.mattermost.com/playbooks/playbooks/h3a39biacpnuim7ufmwiuuoxfo/outline) for the release checklist.
 
 For the desktop app release build process, please refer to [this document](https://developers.mattermost.com/internal/desktop-release-process/).
 
 ## Release timeline
 
 Note: T-minus counts are measured in "working days" (weekdays other than major holidays concurrent in US and Canada) prior to release day.
 
+The Desktop app release schedule follows the cloud/self-hosted release schedule to align all release testing cycles.
+
 **Schedule for Desktop App releases**:
  - Feature Complete deadline is approximately 1 month prior to the release day.
- - Cut release branch based off `master` and cut RC-1 (around T-14).
- - Code Freeze at T-5.
- - Cut Final build at T-2.
-    - [QA approval](https://community.mattermost.com/playbooks/playbooks/h798dt39mpbymb8z5uoiuf4hdo/outline) should be ready by T-2.
+ - Cut release branch based off `master` and cut RC-1 (around T-24).
```

Comment:
I know this is for desktop but since it mentioned it follows the cloud/self-hosted release schedule, I'll comment here anyway.

- In here, RC-1 is mentioned to be cut at T-24.
- At [Synchronized Monthly Releases](https://docs.google.com/document/d/18GyW5VTAspVAqesJFazs1ok_9AQbhoW7IeGge_WL_9M/edit), it didn't exactly mentioned but my interpretation (which I could be totally wrong) was at T-18 when the release qualification starts.
- At [Mattermost Release Proposal](https://docs.google.com/document/d/1ofwGLpjSVKbpQEMZkwftBjabLnpMNBKtfDW4nw23IZw/edit#heading=h.lc7u2vvulv42), RC-1 is cut at T-19 (Judgment day).

Could you clarify when is the actual RC-1 scheduled to be cut?
It would be good to mentioned as well at `operations/research-and-development/product/release-process/feature-release.md`.
cc @lieut-data @toninis

Reply:
@saturninoabril, thanks for bringing attention to this! In the Synchronized Monthly Releases proposal, [T-24: Feature Complete & Release Branch Cut](https://docs.google.com/document/d/18GyW5VTAspVAqesJFazs1ok_9AQbhoW7IeGge_WL_9M/edit#heading=h.1gcwibvy9ggm) just specified the release branch in a bid to avoid features to "slipping in" past feature complete.

When we turn that release branch into RC-1 is probably at the discretion of the delivery team. There is a comment to have "All test servers should be configured to update daily from this branch," but I'm guessing that's independent of when we officially cut RC-1. What's your team's recommendation on this milestone?

Reply:
My understanding is that at T-24, the release branch is cut and there will be build available out of it (ex. `mattermost/mattermost-enterprise-edition:release.9.2`) that could be used for updating test servers until RC becomes available. In my mind, RC-1 should be built right after T-19 (Judgment Day) and should be available at T-18 (Release Qualification). RC-1 is a signal to start release qualification.

Reply:
Makes sense, I've added a note that RC-1 should be cut by the end of T-19.

For desktop and mobile, I added that release branch and RC-1 will be cut by T-19 since those PRs are not normally pre-tested on Community or on release branches, but we can adjust this as needed.